src/components/menu/menu.tsx

```diff
@@ -72,6 +96,15 @@ export class Menu {
     @Prop({ reflect: true })
     public gridLayout = false;
 
+    @Prop()
+    public loadSubItems: SubItemsLoader;
```

Comment:
I get that _internally_ in the component, `loadSubItems` is a nice name, because that's what happens when you call `this.loadSubItems()` (well, _hopefully_ it is, you can never trust them _consumers_ 😝). But for the consumer it's a bit of a weird name for the prop, as it's where you're supposed to provide a callback function.

In `limel-picker`, we've called the property where you're supposed to provide a searcher function, `searcher`, along with a (hopefully) clear description in the docs.

So this property could be called `subItemsLoader` for example, and we'll just have to accept that it looks less neat when we call it inside this component.